pga2d/versors.js

```diff
@@ -77,6 +77,7 @@ export class Motor {
     return `Motor(${this.even})`;
   }
 }
+Motor.ROT90 = Object.freeze(Motor.rotation(Point.ORIGIN, Math.PI / 2));
```

Comment:
Make a note to look through other sketches and use this where needed. 90 degree rotations are pretty common